packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.js

```diff
@@ -1,6 +1,9 @@
-import {api, LightningElement} from 'lwc';
-
+import LOCALE from '@salesforce/i18n/locale';
 import inLabel from '@salesforce/label/c.quantic_InLabel';
+import inclusionFilter from '@salesforce/label/c.quantic_InclusionFilter';
+import inclusionFilter_plural from '@salesforce/label/c.quantic_InclusionFilter_plural';
```

Comment:
The `_zero` label should also be included.

Reply:
oops my bad, just added it, thank you!